compiler/lib/backend/optree/optimizer/transforms/erase_unused_functions.cpp

```diff
@@ -0,0 +1,68 @@
+#include "optimizer/transform.hpp"
+
+#include "compiler/optree/adaptors.hpp"
+#include "compiler/optree/operation.hpp"
+#include "optimizer/opt_builder.hpp"
+#include <deque>
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_set>
```

Comment:
сначала должны быть инклюды из стандартной библиотеки, потом из compiler/..., потом из своей папки (optimizer/opt_builder.hpp), и еще между секциями пустые строки
```suggestion
#include <deque>
#include <map>
#include <memory>
#include <string>
#include <unordered_set>

#include "compiler/optree/adaptors.hpp"
#include "compiler/optree/operation.hpp"

#include "optimizer/opt_builder.hpp"
#include "optimizer/transform.hpp"
```